docs/google-kubernetes-tutorial.md

```diff
@@ -16,127 +17,123 @@ This tutorial takes you through these steps:
 - Connecting to TiDB
 - Shutting down down the Kubernetes cluster (optional)
 
+## Select a Project
 
-## Deploying Kubernetes on Google
-
-TiDB can be deployed onto any Kubernetes cluster: you can bring up a three node cluster however you see fit or look for our documentation on deploying to other systems.
-But here we will bring up a Kubernetes cluster on Google Cloud designed for TiDB.
-Google provides a free small VM called Google Cloud Shell that you can run this tutorial from.
-Just click the button:
-
-[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/pingcap/tidb-operator)
-<!--
-[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/pingcap/tidb-operator&tutorial=docs/google-kubernetes-tutorial.md)
--->
+Please select a project before proceeding.  The approximate cost for running the compute resources in this tutorial is 10 cents/hour.
 
-If you have any issues with this button, you can download this file as markdown and open it.
+<walkthrough-project-billing-setup key="project-id">
+</walkthrough-project-billing-setup>
 
-```sh
-git clone https://github.com/pingcap/tidb-operator
-cd tidb-operator
-teachme docs/google-kubernetes-tutorial.md
-```
+## Enable API Acess
 
-Alternatively, you can run this from your laptop. You just need to [setup the gcloud tool first](https://cloud.google.com/sdk/docs/quickstarts).
+This project will require use of the Compute and Container APIs.  Please enable them before proceeding:
 
+<walkthrough-enable-apis apis="container.googleapis.com,compute.googleapis.com">
+</walkthrough-enable-apis>
 
-## Create a new GCP Project
+## Configure gcloud Defaults
 
-First create a project that this demo will be ran in.
+This step defaults gcloud to your prefered project and [zone](https://cloud.google.com/compute/docs/regions-zones/), which will simplify the commands used for the rest of this tutorial:
 
-	gcloud projects create MY_NEW_PROJECT_NAME
-
-Before you can create a cluster, you must [enable Kubernetes for your project in the console.](https://console.cloud.google.com/projectselector/kubernetes?_ga=2.78459869.-833158988.1529036412)
+	gcloud config set project {{project-id}} &&
```

Comment:
I don't like the double mustaches. MY_PROJECT_ID will still fail if it doesn't exist. I also use angle brackets sometimes. Single mustache would be better.

Reply:
This is called a directive.  It actually substitutes the value from the previous screen :-)  So the user doesn't need to insert their tutorial.